client/src/cb-admin/pages/__tests__/Login.test.js

```diff
@@ -0,0 +1,57 @@
+import {
+  fireEvent,
+  cleanup,
+  waitForElement,
+  wait,
+} from 'react-testing-library';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import renderWithRouter from '../../../tests';
+import Login from '../Login';
+
+
+describe('Login Component', () => {
+  afterEach(cleanup);
+
+  test(':: incorrect user details returns 401 and displays error message', async () => {
+    const mock = new MockAdapter(axios);
```

Comment:
Do we need to create a new instance of the mock adapter for each test case? Can we just re-use it?